src/App.js

```diff
@@ -9,6 +9,26 @@ import {
 import AppHeader from './components/AppHeader';
 import Home from './pages/Home';
 import PostsManager from './pages/PostsManager';
+import { init, AuthType, LiveboardEmbed, EmbedEvent } from '@thoughtspot/visual-embed-sdk';
+import PostsList from './pages/PostsList';
+
+
+init({
+    thoughtSpotHost: "https://embed-1-do-not-delete.thoughtspotdev.cloud",
+    authType: AuthType.TrustedAuthToken,
+    username: "yuanyuan.liu",
```

Comment:
Username should be extracted from the okta login payload. It should be the same username the user logged in with into Okta.